config/prometheus/rbac-lvm-op-metric-role-n-rolebindings.yaml

```diff
@@ -0,0 +1,33 @@
+# Reference link: https://kubernetes.io/docs/reference/access-authn-authz/rbac/
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: lvm-operator-metrics
+  namespace: system
+rules:
+- apiGroups:
+  - "" # "" indicates the core API group
+  resources: 
+    - services
```

Comment:
can you please detab contents under `resources` and `verbs`, same for rest of manifests as well?